packages/plugins/apollo-usage-report/src/index.ts

```diff
@@ -121,15 +171,17 @@ export function useApolloUsageReport(options: ApolloUsageReportOptions = {}): Pl
             );
           }
         },
+
         onSchemaChange({ schema }) {
           if (schema) {
-            schemaIdSet$ = handleMaybePromise(
-              () => hashSHA256(printSchema(schema), yoga.fetchAPI),
-              id => {
-                schemaId = id;
+            schemaIdSet$ = hashSHA256(printSchema(schema), yoga.fetchAPI)
```

Comment:
Maybe we can use `printSchemaWithDirectives` instead to print the directives because supergraphs heavily use it.